Index: ../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/app/src/test/java/com/example/androidexample/ExampleUnitTest.java
===================================================================
diff --git a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/app/src/test/java/com/example/androidexample/ExampleUnitTest.java b/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/app/src/test/java/com/example/androidexample/ExampleUnitTest.java
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/app/src/test/java/com/example/androidexample/ExampleUnitTest.java	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,17 +0,0 @@
-package com.example.androidexample;
-
-import org.junit.Test;
-
-import static org.junit.Assert.*;
-
-/**
- * Example local unit test, which will execute on the development machine (host).
- *
- * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
- */
-public class ExampleUnitTest {
-    @Test
-    public void addition_isCorrect() {
-        assertEquals(4, 2 + 2);
-    }
-}
\ No newline at end of file
Index: ../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/gradle/wrapper/gradle-wrapper.properties
===================================================================
diff --git a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/gradle/wrapper/gradle-wrapper.properties b/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/gradle/wrapper/gradle-wrapper.properties
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/gradle/wrapper/gradle-wrapper.properties	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,6 +0,0 @@
-#Thu Sep 28 12:08:05 CDT 2023
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0-bin.zip
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
Index: ../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/build.gradle
===================================================================
diff --git a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/build.gradle b/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/build.gradle
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/build.gradle	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,4 +0,0 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-plugins {
-id 'com.android.application' version '8.0.0' apply false
-}
Index: ../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/settings.gradle
===================================================================
diff --git a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/settings.gradle b/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/settings.gradle
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/AndroidExample/settings.gradle	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,17 +0,0 @@
-pluginManagement {
-    repositories {
-        google()
-        mavenCentral()
-        gradlePluginPortal()
-    }
-}
-dependencyResolutionManagement {
-    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
-    repositories {
-        google()
-        mavenCentral()
-    }
-}
-
-rootProject.name = "AndroidExample"
-include ':app'
Index: ../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/README.md
===================================================================
diff --git a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/README.md b/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/README.md
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp4/tutorials-android_unit1_3_login_signup/tutorials-android_unit1_3_login_signup/README.md	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,10 +0,0 @@
-# android_unit1_3
-
-### What is this example: 
-
-A "login & signup" example. This is only an example of the interface, NO actual connections are made in this example.
-
-### version tested
-|Android Studio            | Android SDK | Gradle | Emulator |
-|--------------------------|-------------|--------|----------|
-|Giraffe 2022.3.1 Patch 2  |33           | 8.0.0  | Pixel 3a |
Index: ../../Experiments/Nick127k/Exp5/LoginScreen2/app/build/intermediates/packaged_res/debug/drawable/roundstyle.xml
===================================================================
diff --git a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/build/intermediates/packaged_res/debug/drawable/roundstyle.xml b/../../Experiments/Nick127k/Exp5/LoginScreen2/app/build/intermediates/packaged_res/debug/drawable/roundstyle.xml
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/build/intermediates/packaged_res/debug/drawable/roundstyle.xml	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,4 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <corners android:radius="18dp"/>
-</shape>
\ No newline at end of file
Index: ../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/java/com/example/loginscreen/MainActivity3.java
===================================================================
diff --git a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/java/com/example/loginscreen/MainActivity3.java b/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/java/com/example/loginscreen/MainActivity3.java
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/java/com/example/loginscreen/MainActivity3.java	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,14 +0,0 @@
-package com.example.loginscreen;
-
-import androidx.appcompat.app.AppCompatActivity;
-
-import android.os.Bundle;
-
-public class MainActivity3 extends AppCompatActivity {
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_main3);
-    }
-}
\ No newline at end of file
Index: ../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/alegreya_sc_bold_italic.xml
===================================================================
diff --git a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/alegreya_sc_bold_italic.xml b/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/alegreya_sc_bold_italic.xml
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/alegreya_sc_bold_italic.xml	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<font-family xmlns:app="http://schemas.android.com/apk/res-auto"
-        app:fontProviderAuthority="com.google.android.gms.fonts"
-        app:fontProviderPackage="com.google.android.gms"
-        app:fontProviderQuery="name=Alegreya SC&amp;weight=700&amp;italic=1"
-        app:fontProviderCerts="@array/com_google_android_gms_fonts_certs">
-</font-family>
Index: ../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/arima_madurai.xml
===================================================================
diff --git a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/arima_madurai.xml b/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/arima_madurai.xml
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/font/arima_madurai.xml	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<font-family xmlns:app="http://schemas.android.com/apk/res-auto"
-        app:fontProviderAuthority="com.google.android.gms.fonts"
-        app:fontProviderPackage="com.google.android.gms"
-        app:fontProviderQuery="Arima Madurai"
-        app:fontProviderCerts="@array/com_google_android_gms_fonts_certs">
-</font-family>
Index: ../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml
===================================================================
diff --git a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml b/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml
deleted file mode 100644
--- a/../../Experiments/Nick127k/Exp5/LoginScreen2/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
+++ /dev/null	(revision 9e67ecd32592d09e5791bdd2de046a8d3e208e3a)
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
-    <monochrome android:drawable="@drawable/ic_launcher_foreground" />
-</adaptive-icon>
\ No newline at end of file
diff --git a/../../.gitignore b/../../.gitignore
deleted file mode 100644
